code42_connector.py

```diff
@@ -154,16 +164,83 @@ def _handle_list_high_risk_employees(self, param, action_result):
         action_result.update_summary({"total_count": total_count})
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    """ USER ACTIONS """
+
+    @action_handler_for("create_user")
+    def _handle_create_user(self, param, action_result):
+        username = param["username"]
+        password = param.get("password") or _generate_password()
+        response = self._client.users.create_user(
+            org_uid=param["org_uid"],
+            username=username,
+            email=username,
+            password=password,
+            first_name=param.get("first_name"),
+            last_name=param.get("last_name"),
+            notes=param.get("notes"),
+        )
+        user_id = response["userUid"]
+        action_result.add_data(response.data)
+        action_result.update_summary({"user_id": user_id, "username": username})
```

Comment:
Something we noticed:  If you include a summary and a status message, only the status message shows up in the UI, but if you leave off the status message, the summary shows up. There may be other places where it's visible but I have yet to find it.